README.md

```diff
@@ -18,29 +18,29 @@ Usage
 Run `ttv --help` to get help, or infer what you can from one of these examples:
 
     # Split CSV file into two sets of a fixed number of rows
-    $ ttv split data.csv --rows=train=9000 --rows=test=1000 --uncompressed-input
+    $ ttv split data.csv --rows=train=9000 --rows=test=1000
 
     # Accepts gzipped data (no flag required). Shorthand argument version. As many splits as you like!
     $ ttv split data.csv.gz --rows=train=65000,validation=15000,test=15000
```

Comment:
```suggestion
    $ ttv split data.csv.gz --rows=train=65000,validation=15000,test=15000 -d
```

It was 4 years ago but it looks like I inverted the flag so it's required for gzipped data and changed from `-u` to `-d`.